### PR TITLE
[USER]: pagination added to index close #195

### LINKED ISSUE #195: Add pagination to users index
We should add pagination to the user's index so we don't just get `.all`.

### PR DESCRIPTION
**Pull request by snewcomer**
Added pagination to user index controller

### CODE REVIEW

File: test/controllers/user_controller_test.exs [hunk 1]
```diff
@@ -268,4 +268,60 @@ defmodule CodeCorps.UserControllerTest do
       refute json["valid"]
     end
   end
+
+  describe "pagination" do
+    test "specifying a page size works", %{conn: conn} do
+      insert(:user)
+      insert(:user)
+      insert(:user)
```
Comment (begedin) [inline test/controllers/user_controller_test.exs]: There's an `insert_list(3, :user)`

File: test/controllers/user_controller_test.exs [hunk 2]
```diff
@@ -268,4 +268,60 @@ defmodule CodeCorps.UserControllerTest do
...
+        |> json_response(200)
+
+      assert json["data"] |> Enum.count == 2
+    end
+
+    test "specifying a page number works", %{conn: conn} do
+      insert(:user)
+      insert(:user)
+      user_to_test = insert(:user)
+      insert(:user)
```
Comment (begedin) [inline test/controllers/user_controller_test.exs]: could go with `user_to_test = insert_list(4, :user) |> List.last`

and then test the last element of the data below.

**Comment by begedin** [comment]
Looks good after a few minor tweaks.

**Comment by npendery** [comment]
LGTM 👍

**Review by joshsmith — APPROVED** [review]

**Pull request merged**